lib/gitlab/ci/templates/pipeline/AndroidTemplate.yml

```diff
@@ -73,6 +75,48 @@ deployDebugApk:
         https://slack.com/api/files.upload
   cache: {}
 
+testRelease:
```

Comment:
Can you please add a bullet to the 1.3.0 Change Log about adding the test release, assemble release apk, and deploy release apk jobs to the Android Pipeline Template? https://github.com/chesapeaketechnology/gitlab-templates/blob/release/1.3/README.md#130-on-future-date--official-stable-release-with-many-changes